docs/asm-installation-manifests/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Sample to illustrate how to customize the ASM channel, you need to replace the variables ASM_VERSION and ASM_CHANNEL in for-asm-channel/kustomization.yaml before deploying this file.
+# As an example: sed -i "s/ASM_VERSION/asm-managed-rapid/g;s/ASM_CHANNEL/rapid/g" for-asm-channel/kustomization.yaml
```

Comment:
Good point about MacOS, so typically replacing `""` by `''`, right?

Reply:
Done.

Reply:
After some tests, `''` doesn't work on Linux/Cloud Shell if there is a bash variable in there.

After chatting about that with TWs, it appears that because we are targeting Cloud Shell, we are fine with keeping this `sed -i ""...` approach.

With that said, that's really good to know for MacOS use case, thanks for raising this.

Reply:
I think Shabir might be talking about the need for an extra pair of quotations (as described in [this StackOverflow post](https://unix.stackexchange.com/questions/401905/bsd-sed-vs-gnu-sed-and-i/401928#401928)).

But no changes necessary since Nicola (TW) said we should be encouraging (and primarily supporting) Cloud Shell anyway.

We can resolve this thread IMO.